files/private-chef-cookbooks/private-chef/templates/default/pivotal.rb.erb

```diff
@@ -1,4 +1,5 @@
 node_name "pivotal"
 chef_server_url "<%= @prefix %>://<%= @vip %>"
 chef_server_root "<%= @prefix %>://<%= @vip %>"
-client_key "/etc/opscode/pivotal.pem"
\ No newline at end of file
+client_key "/etc/opscode/pivotal.pem"
+ssl_verify_mode :verify_none
```

Comment:
Set ssl_verify_mode to verify_none for pivotal.rb knife config.

Chef 12 turns on SSL verification by default. This sets it back to how it was before we pulled in Chef 12.

We generate certs for the fqdn and this config talks to the vips. The vips are always localhost (unless someone did some serious messing around) so it should be fine to always trust that the localhost we are talking to is the correct one.